src/pages/NIHicWebform.js

```diff
@@ -0,0 +1,2502 @@
+import {Component} from 'react';
+import {RadioButton} from '../components/RadioButton';
+import {a, br, div, fieldset, form, h, h3, hr, input, label, span, textarea} from 'react-hyperscript-helpers';
+import {Link} from 'react-router-dom';
+import Select from 'react-select';
+import {Alert} from '../components/Alert';
+import {Notification} from '../components/Notification';
+import {PageHeading} from '../components/PageHeading';
+import {DAR} from '../libs/ajax';
+import * as fp from 'lodash/fp';
+import AsyncSelect from 'react-select/async';
+
+class NIHICWebform extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      dacList: [],
+      selectedDac: {},
+      allDatasetNames: [],
+      updateDataset: {},
+      disableOkBtn: false,
+      showValidationMessages: false,
+      formData: {
+        methods: '',
+        genetic: '',
+        publication: '',
+        collaboration: '',
+        ethics: '',
+        geographic: '',
+        moratorium: '',
+        forProfit: '',
+        hmb: false,
+        poa: false,
+        diseases: false,
+        ontologies: [],
+        other: false,
+        otherText: '',
+        generalUse: false
+      },
+      datasetData: {
+        datasetName: '',
+        researcher: '',
+        datasetRepoUrl: '',
+        dataType: '',
+        species: '',
+        phenotype: '',
+        description: '',
+        dac: '',
+        publicAccess: false,
+      },
+      problemSavingRequest: false,
+      problemLoadingUpdateDataset: false,
+      submissionSuccess: false,
+    };
+  };
+
+  handleChange = (e) => {
+    const field = e.target.name;
+    const value = e.target.value;
+    this.setState(prev => {
+      prev.datasetData[field] = value;
+      prev.disableOkBtn = false;
+      prev.problemSavingRequest = false;
+      prev.submissionSuccess = false;
```

Comment:
Since this is a demo page, we should remove all traces of trying to save anything.